machine.md

```diff
@@ -4,6 +4,41 @@ Finetuning different aspects of the hardware which are not device related
 (BIOS, mainboard, ...) is sometimes necessary to allow Guest Operating Systems
 to properly boot and reboot.
 
+## Machine Type
+
+QEMU is able to work with two different classes of chipsets for x86\_64, so
+called machine types. The x86\_64 chipsets are i440fx (also called pc) and q35.
+They are versioned based on qemu-system-$ARCH, following the format
+`pc-${machine_type}-${qemu_version}`, e.g. `pc-i440fx-2.10` and `pc-q35-2.10`.
+
+KubeVirt defaults to QEMU's newest q35 machine type. If custom machine type is
```

Comment:
Nit: "If a custom machine type is desired"

Reply:
so lets fix this one type and then merge this.